service/src/test/java/uk/gov/hmcts/reform/fpl/service/FeatureToggleServiceTest.java

```diff
@@ -196,22 +156,17 @@ void shouldMakeCorrectCallForAllocatedJudgeNotificationC2Application(Boolean tog
         givenToggle(toggleState);
 
         assertThat(service.isAllocatedJudgeNotificationEnabled(C2_APPLICATION)).isEqualTo(toggleState);
-
-        verify(ldClient).boolVariation(eq("judge-notification"), ldUser.capture(), eq(false));
-        assertThat(ldUser.getValue().getCustomAttributes()).isEqualTo(
-            getLdUserWithAllocatedJudgeKey().getCustomAttributes());
+        verify(ldClient).boolVariation(eq("judge-notification"), any(LDUser.class), eq(false));
     }
 
     @ParameterizedTest
     @MethodSource("userAttributesTestSource")
-    void shouldNotCarryAttributesBetweenRequests(Runnable functionToTest, Runnable accumulateFunction,
+    void shouldNotAccumulateAttributesBetweenRequests(Runnable functionToTest, Runnable accumulateFunction,
                                                  List<UserAttribute> attributes) {
         accumulateFunction.run();
         functionToTest.run();
 
         verify(ldClient, times(2)).boolVariation(anyString(), ldUser.capture(), anyBoolean());
-
-        assertThat(ldUser.getValue().getCustomAttributes()).containsExactlyInAnyOrderElementsOf(attributes);
```

Comment:
Ahh, we need this, might have got confused in the chat

Reply:
My bad, got a bit delete happy :)